app/src/main/java/com/mapbox/maps/testapp/examples/StyleCirclesCategoricallyActivity.kt

```diff
@@ -57,46 +57,13 @@ class StyleCirclesCategoricallyActivity : AppCompatActivity() {
               get {
                 literal("ethnicity")
               }
-              stop {
-                literal("white")
-                rgb {
-                  literal(251)
-                  literal(176)
-                  literal(59)
-                }
-              }
-              stop {
-                literal("Black")
-                rgb {
-                  literal(34)
-                  literal(59)
-                  literal(83)
-                }
-              }
-              stop {
-                literal("Hispanic")
-                rgb {
-                  literal(229)
-                  literal(94)
-                  literal(94)
-                }
-              }
-              stop {
-                literal("Asian")
-                rgb {
-                  literal(59)
-                  literal(178)
-                  literal(208)
-                }
-              }
-              stop {
-                literal("Other")
-                rgb {
-                  literal(204)
-                  literal(204)
-                  literal(204)
-                }
-              }
+              // use stop expressions as input, output format. where first expression is stop input
+              // and second expression is stop output for circleColor.
+              stop("white") { rgb(251.0, 176.0, 59.0) }
+              stop("Black") { rgb(34.0, 59.0, 83.0) }
+              stop("Hispanic") { rgb(229.0, 94.0, 94.0) }
+              stop("Asian") { rgb(59.0, 178.0, 208.0) }
+              stop("Other") { rgb(204.0, 204.0, 204.0) }
```

Comment:
em.. this looks a bit weird.. wdyt cc @tobrun

Reply:
Yeah, i tried `stop("white", rgb(251.0, 176.0, 59.0))`, but kotlin always simplifies if last parameter is lambda. :D

Reply:
imho it looks fine